terraform/modules/rds/main.tf

```
@@ -0,0 +1,56 @@
+# Non-compliant RDS configuration (missing encryption, backups, and monitoring)
+resource "aws_db_instance" "main" {
+  identifier        = "${var.environment}-db"
+  engine           = "postgres"
+  engine_version   = "13.7"
+  instance_class   = var.instance_class
+  allocated_storage = 20
+
+  # Non-compliant: Credentials in plain text
+  name     = var.db_name
```

The RDS instance is using hardcoded credentials, which is a security risk. Credentials should be managed securely using AWS Secrets Manager or SSM Parameter Store.

```suggestion:-0+0
username = var.db_username
password = var.db_password
```